imap_processing/lo/l1c/lo_l1c.py

```diff
@@ -484,3 +491,98 @@ def create_datasets(
             )
 
     return dataset
+
+
+def set_background_rates(
+    pointing_start_met: float,
+    pointing_end_met: float,
+    species: FilterType,
+    anc_dependencies: list,
+    attr_mgr: ImapCdfAttributes,
+) -> xr.DataArray:
+    """
+    Set the background rates for the specified species.
+
+    The background rates are set to a constant value of 0.01 counts/s for all bins.
+
+    Parameters
+    ----------
+    pointing_start_met : float
+        The start MET time of the pointing.
+    pointing_end_met : float
+        The end MET time of the pointing.
+    species : FilterType
+        The species to set the background rates for. Can be "h" or "o".
+    anc_dependencies : list
+        Ancillary files needed for L1C data product creation.
+    attr_mgr : ImapCdfAttributes
+        Attribute manager used to get the L1C attributes.
+
+    Returns
+    -------
+    background_rates : xarray.DataArray
+        The background rates for the specified species.
+    """
+    if species not in {FilterType.HYDROGEN, FilterType.OXYGEN}:
+        raise ValueError("Species must be 'h' or 'o'.")
+
+    bg_rates = np.zeros((7, 3600, 40), dtype=np.float16)
+    bg_stat_uncert = np.zeros((7, 3600, 40), dtype=np.float16)
+    bg_sys_err = np.zeros((7, 3600, 40), dtype=np.float16)
+
+    # read in the background rates from ancillary file
+    if species == FilterType.HYDROGEN:
+        background_df = lo_ancillary.read_ancillary_file(
+            next(s for s in anc_dependencies if "hydrogen-background" in s)
```

Comment:
I'm wondering if rather than passing in a list of ancillary files we should pass in the `ProcessingInputCollection` directly so that you could do the filtering through that class.

```
dependencies.get_file_paths(source="lo", descriptor="hydrogen-background")[0]
```